src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -86,6 +86,8 @@ abstract class AbstractCloner implements ClonerInterface
         'Symfony\Component\HttpClient\Response\CurlResponse' => ['Symfony\Component\VarDumper\Caster\SymfonyCaster', 'castHttpClientResponse'],
         'Symfony\Component\HttpClient\Response\NativeResponse' => ['Symfony\Component\VarDumper\Caster\SymfonyCaster', 'castHttpClientResponse'],
         'Symfony\Component\HttpFoundation\Request' => ['Symfony\Component\VarDumper\Caster\SymfonyCaster', 'castRequest'],
+        'Symfony\Component\Uid\Ulid' => ['Symfony\Component\VarDumper\Caster\SymfonyCaster', 'castUlid'],
+        'Symfony\Component\Uid\Uuid' => ['Symfony\Component\VarDumper\Caster\SymfonyCaster', 'castUuid'],
```

Comment:
why not map both to `castUid(AbstractUid $uid)`?

Reply:
That's what I did first but I think it's easier to read if it's copy/pasted because there are no conditions (the formats order is different + the datetime format too). I wanted to follow the same order I did in the inspect commands.

Reply:
why using a different timestamp format ?

Reply:
Because ULIDs are "less" precise than UUIDs (milliseconds vs 0.1 microseconds).